Makefile

```diff
@@ -107,8 +107,8 @@ test-services-docker:
 .PHONY: test-cluster
 test-cluster: build install
 	-zapi rm files
-	-aws s3 rm --recursive $(ZQD_DATA_URI)
-	zapi new -k archivestore -d $(ZQD_DATA_URI) files
+	-aws s3 rm --recursive $(ZQD_DATA_URI)/zd
```

Comment:
I would just do `-zapi rm files` instead of the direct aws s3 cmd

Reply:
The "s3 rm" is needed because the "zapi postpath" below will add more S3 files to that directory -- duplicate records from what was added with previous tests by "zapi postpath".

"zapi rm files" has no effect on S3, so both commands are required.